.gitignore

```diff
@@ -39,9 +38,7 @@ stats.html
 
 *.log
 .DS_Store
-node_modules
```

Comment:
can you please leave node_modules in gitignore

Reply:
Ah, I should have left a comment explaining the change. You can check duplicate node_modules in `.gitignore`. so I removed it. https://github.com/TanStack/query/pull/5813/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947L5

removing `dist` have same reason